index.html

```diff
@@ -0,0 +1,61 @@
+        </section>
+        <section>
+            <div id="careers">
+                <h2>Careers I've had.</h2>
+                <ul id="list1">
+                    <li>Logistician</li>
+                    <li>Heavy Equipment Operator</li>
+                    <li>New and Used Car Salesman</li>
+                    <li>Heavy Truck Delivery Driver</li>
+                    <li>Car Crusher</li>
+                </ul>
+            </div>
+        </section>
+        <section>
+            <div id="accomplish">
+                <h3>My Lifes's Biggest Accomplishments</h3>
+                <ol id="list2">
+                    <li>Obtained the Rank of Eagle Scout</li>
+                    <li>Got Married</li>
+                    <li>Qualified for the Olympics</li>
+                    <li>Built the Fastest Car of its Type in the Region</li>
+                </ol>
+            </div>
+        </section>
+        <section>
+            <p id="power"><span style="text-decoration: underline;">Anyone can do anything. Don't let others tell you not.</span></p><br></br>
+            <div>
+                <button id="quizbutton">Take Quiz</button>
+            </div><br></br>
+        </section>
+        <section id="results">
+            <h2><em>So, how well do you know me? Take the quiz above to find out. Your results will show below so be sure to check when done.</em></h2>
+            <p id="total"></p>
+        </section>
+    </main>
+    <footer>This quiz was brought to you by CodyCoder</footer>
+    <script type="module" src="app.js"></script>
+</body>
+
```

Comment:
Everything looks good. In the future try to leave empty lines between each section to make them easier to tell apart at a glance.